string name
float32 attention_value
float32 boost_value
asmo/MessageAction[] message_actions
